.github/workflows/greetings.yml

```diff
@@ -12,5 +12,5 @@ jobs:
       - uses: actions/first-interaction@v1
```

Comment:
meh, i don't see how this is needed

Reply:
breh i meant the changes that you made to the file sorry

Reply:
Is needed.

Reply:
nah, it's not really necessary. also I figured out most of these changes so ill just PR these into the main one

Reply:
> nah, it's not really necessary. also I figured out most of these changes so ill just PR these into the main one

Okay.